ProcessLib/InitialCondition.cpp

```diff
@@ -21,22 +21,36 @@
 namespace ProcessLib
 {
 std::unique_ptr<InitialCondition> createUniformInitialCondition(
-    BaseLib::ConfigTree const& config, int const /*n_components*/)
+    BaseLib::ConfigTree const& config, int const n_components)
 {
     config.checkConfParam("type", "Uniform");
 
-    auto value = config.getConfParam<double>("value");
```

Comment:
Why not keep "value" as the option for single-component process variables and only use "values" for multi-component?